Matrix.h

```diff
@@ -702,6 +705,37 @@ class Matrix
       void
       inverse();
 
+      /**
+       * @brief Returns a column of the matrix.
+       *
+       * @return A column of the matrix.
+       */
```

Comment:
Maybe comment that the returned vector needs to be deleted by the user (not owned by `Matrix`).